LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/resources/outbound-before-render.html

```diff
@@ -5,7 +5,7 @@
     navigation: auto;
   }
 </style>
-<script id="blocker" async src="common.js?pipe=trickle(d10)" blocking="render"></script>
+<link id="blocker" rel="expect" href="#target" blocking="render">
```

Comment:
This doesn't work, it only passed due to https://github.com/WebKit/WebKit/pull/33163 being unfixed.

I'm chatting with the spec authors to get the right behaviour clarified.